spec/features/availability_spec.rb

```diff
@@ -321,6 +321,9 @@
         end
       end
 
+      it 'has the correct ILL url if a record has a holding in a CHECKEDOUT location'
+      it 'has the correct ILL url if a record has a holding in a reserves scan location'
```

Comment:
These are stubbed because we want to rush this change and get the specs in later?

Reply:
I was also thinking adding after we refactor these specs to use Sinatra, because you know items changing locations causing specs to fail right now.

Reply:
perfect sounds good